.mocharc.yml

```
@@ -0,0 +1,10 @@
+diff: true
+extension: ['js']
+package: './package.json'
+slow: '75'
+spec: 'test/**/*.js'
+timeout: '2000'
+ui: 'bdd'
+watch-files:
+  - 'src/**/*.js'
+  - 'test/**/*.js'
```

Can we configure to include the test for the ua app?

```suggestion
  - 'test/**/*.js'
  - 'ua/src/**/*.js'
  - 'ua/test/**/*.js'
```